src/core_modules/capture-core/components/Pages/ViewEvent/EventDetailsSection/EventDetailsSection.container.js

```diff
@@ -8,6 +8,7 @@ import {
 
 const mapStateToProps = (state: ReduxState) => ({
     showEditEvent: state.viewEventPage.eventDetailsSection && state.viewEventPage.eventDetailsSection.showEditEvent,
+    programId: state.currentSelections.programId,
```

Comment:
Hey @simonadomnisoru,
Can't really remember, but I have a vague memory that we wanted to move away from `currentSelections` as this might not be fully trusted to be update. This was the reason we removed the `connected-redux-router`-dependency.
Is this a concern that we're still having or is it okay to use in this case?

Reply:
Hey @eirikhaugstulen,

Indeed, we try to move away from `currentSelections`. But for this page and use case, I believe is safe to use it. This is because in a [previous commit](https://github.com/dhis2/capture-app/commit/0f23429c9a2a9e171e6e3ae1003d1008abb793bd#diff-3716f21e7b006a1c4cd91722cec7a4ae7ad50395c60ef78a2f9fb0dc30dcd560L32), I remove it from the epics chain. Now by adding it again I fix the bug that was introduced by the above commit. 

Thanks!

Reply:
All fine by me, just wanted to check in! Looks good 👍